Source/Make.WarpX

```diff
@@ -71,6 +71,20 @@ ifeq ($(USE_OPENBC_POISSON),TRUE)
   DEFINES += -DFFT_FFTW -DMPIPARALLEL -DUSE_OPENBC_POISSON
 endif
 
+ifeq ($(USE_OPENPMD), TRUE)
+   OPENPMD_LIB_PATH ?= NOT_SET
+   ifneq ($(OPENPMD_LIB_PATH),NOT_SET)
+     LIBRARY_LOCATIONS += $(OPENPMD_LIB_PATH)
+   endif
```

Comment:
Should we return an error if `USE_OPENPMD=TRUE` but `OPENPMD_LIB_PATH` is not defined? Something like
```
   else
     $(error MAKE ERROR: OPENPMD_LIB_PATH is not defined (required for USE_OPENPMD=TRUE))
```
This is also valid for `OPENPMD_INCLUDE_PATH` and FFTW things.

Reply:
For FFTW, we should not make  it an error because the compiler wrapper on many machines knows how to handle it.  Same could be true for openpmd.

Reply:
Agreed. If openPMD-api is installed in a system path or appropriate environment variables are set, e.g. by modules, a missing explicit path hint is not required.